src/module.ts

```diff
@@ -63,30 +63,28 @@ export namespace TypeRef {
   // special types
   export const None: TypeRef = 0 /* _BinaryenTypeNone */;
   export const Unreachable: TypeRef = 1 /* _BinaryenTypeUnreachable */;
+  export const Auto: TypeRef = -1 /* _BinaryenTypeAuto */;
```

Comment:
```suggestion
  export const Auto: TypeRef = -1 /* _BinaryenTypeAuto */;
  export const None: TypeRef = 0 /* _BinaryenTypeNone */;
  export const Unreachable: TypeRef = 1 /* _BinaryenTypeUnreachable */;
```

Reply:
Figured that we are not really using the `Auto` type, and given that it's always better to be explicit, I removed it.